realm/src/main/java/io/realm/Realm.java

```diff
@@ -895,6 +904,80 @@ public void writeCopyTo(File destination) throws IOException {
         return result;
     }
 
+    /**
+     * Converts a RealmObject that has been created using it's standard constructors instead of
```

Comment:
I would word it something like:
Copies a RealmObject to the Realm instance and returns the copy.
It is important to notice that changes applied to the original RealmObject will not be reflected in the Realm copy.